src/util.ts

```diff
@@ -166,3 +167,32 @@ export function formatAsUTCISO(
 
   return resultString;
 }
+
+export class PassThroughShim extends PassThrough {
+  private shouldEmitReading = true;
+  private shouldEmitWriting = true;
+
+  constructor() {
+    super();
+  }
```

Comment:
[Optional] No need for the constructor here - the [default constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor) covers this.

Reply:
Removed